Mathlib/RingTheory/RingHom/Etale.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2025 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.RingTheory.RingHom.Smooth
+import Mathlib.RingTheory.RingHom.Unramified
+
+/-!
+# Etale ring homomorphisms
+
+We show the meta properties of étale morphisms.
+-/
+
+universe u
+
+namespace RingHom
+
+variable {R S : Type u} [CommRing R] [CommRing S]
+
+/-- A ring hom `R →+* S` is etale, if `S` is an etale `R`-algebra. -/
+@[algebraize Algebra.Etale]
+def Etale {R S : Type u} [CommRing R] [CommRing S] (f : R →+* S) : Prop :=
+  letI := f.toAlgebra
+  Algebra.Etale R S
+
+variable {R S : Type u} [CommRing R] [CommRing S] (f : R →+* S)
+
+lemma etale_algebraMap [Algebra R S] : (algebraMap R S).Etale ↔ Algebra.Etale R S := by
+  simp only [RingHom.Etale]
+  congr!
+  exact Algebra.algebra_ext _ _ fun _ ↦ rfl
+
+lemma etale_iff_formallyUnramified_and_smooth : f.Etale ↔ f.FormallyUnramified ∧ f.Smooth := by
+  algebraize [f]
+  simp only [Etale, Smooth, FormallyUnramified]
+  refine ⟨fun h ↦ ⟨inferInstance, ?_⟩, fun ⟨h1, h2⟩ ↦ ⟨?_, inferInstance⟩⟩
+  · constructor <;> infer_instance
+  · rw [Algebra.FormallyEtale.iff_unramified_and_smooth]
+    constructor <;> infer_instance
+
+lemma Etale.eq_and :
```

Comment:
I think there are multiple possible "`and`"s that etale could be, so I think this should be spelled out as `eq_formallyUnramified_and_smooth`.